Leave/leave_interface.py

```diff
@@ -53,24 +52,31 @@ async def HandleLeaveReactions(client, payload):
     message = await channel.fetch_message(payload.message_id)
     embed = message.embeds[0]
     
-    status = UI.ParseEmoji(payload.emoji)
-    if status == None:
+    Action = UI.ParseEmoji(payload.emoji)
```

Comment:
Any reason to have `Action` vs `action` here?

Reply:
no not at all it was a mistake

Reply:
Done